build/composer-php-72.json

```diff
@@ -5,10 +5,9 @@
     "license": "MIT",
     "keywords": ["static analysis", "phpstan-extension"],
     "require": {
-        "php": "^7.2 || 8.0.*",
+        "php": "^7.2 || ^8.0",
```

Comment:
doesn't this change in requirements mean, that everyone will get the php 7.2 downgraded version?

before this PR everyone using PHP 8.1/8.2 got the `0.2.0` tag while only people using `^7.2 || 8.0.*` would have gotten the `0.2.0.72` tag

Reply:
Indeed. This is a solution parallel to removal nette/utils.

Now there is a question,  if the downgraded is needed after all. I think we can remove it now too

Reply:
you mean you no longer need to publish 2 different artifacts, because using the downgraded for everyone is good enough?

or do you mean dropping the 7.2 downgrading and use a higher min-php version?

Reply:
> or do you mean dropping the 7.2 downgrading

...and using PHP 7.2+ and 8+ requirement, as all the dependencies have it :)

Reply:
Oh, but that would mean we have to code in PHP 7.2 too, right :D